nss-day-cohort-48/fun-website-1-digital-drops
# changed the index.html contact info and @imported the links.css

-edited the html footer so the contact info is uniformed across the site.
-did an @import at the top of the main.css file for the links.css

to test:
1. clone and pull request
2. code  . and serve
3. verify that the contact section in the index.html page is the same as the three other pages.
